### ISSUE TITLE
Support for multibyte encoded strings in output

### ISSUE THREAD

**Issue by gcampbell-msft** [microsoft/vscode-serial-monitor#90]
Support for muti-byte encoded strings in the Serial Monitor output. 

See #88 for more context.

**Comment by pkoevesdi** [comment]
I come here, because I see this annoying as well:
![grafik](https://github.com/microsoft/vscode-serial-monitor/assets/33196971/7f6b31ef-b959-448f-a303-e57daffb9368)
Sent Text ist: "Bitte Menüpunkt wählen:"

**Comment by gcampbell-msft** [comment]
@pkoevesdi Thanks for the update. This is almost certainly because of those characters having multibyte encodings. I'm not sure when we'll get to this feature, but please upvote the original post so that we can know people are interested in it. Thanks!

**Comment by pkoevesdi** [comment]
Sorry, but to me, it should be no question of interest, but a naturalness do deal with extended characters. I mean, it's 2024, not 1988 ;-). And to my knowledge, this is not a 2-byte-character, it's ASCII 252dec.

**Comment by pkoevesdi** [comment]
Nope, sorry, the hex output shows it as c3 bc. But in the soruce code of the sender device I put it as a "ü", like here, which is supposed to be Ascii 252.

**Comment by pkoevesdi** [comment]
The thing is, I could easily live with a 1-byte-character set, ISO-8859-1. But I do my stuff in rust, and the compiler doesn't accept ISO-8859-1, it wants UTF-8 sources. So, I expect my serial monitor to render UTF-8 correctly (which by the way several serial programs do, but unfortunately not all).

**Comment by gcampbell-msft** [comment]
@pkoevesdi Thanks for the responses. To confirm, you're saying that you've confirmed that the issue is coming from multi-byte encodings?

**Comment by pkoevesdi** [comment]
Yes, in this case UTF-8 (multi-byte). ISO 8859-1 (1-byte) works.
